README_cn.md

```diff
@@ -43,24 +43,45 @@ HAMi 是[Cloud Native Computing Foundation](https://cncf.io/)(CNCF)基金会的s
 HAMi通过支持设备共享和设备资源隔离，为包括GPU在内的多个异构设备提供设备虚拟化。有关支持设备虚拟化的设备列表，请参阅 [支持的设备]（#支持设备）
 
 ### 设备复用能力
-
+HAMi支持:
 - 允许通过指定显存来申请算力设备
-- 算力资源的硬隔离
 - 允许通过指定算力使用比例来申请算力设备
 - 对已有程序零改动
 
 <img src="./imgs/example.png" width = "500" /> 
 
+```yaml
+# pod-A of UserA
+      resources:
+        limits:
+          nvidia.com/gpu: 2 # requesting 2 vGPUs
+          nvidia.com/gpumem: 10000 # Each vGPU containers 10G device memory
+          # nvida.com/gpucores: 50 # Utilization limit of this vGPU is set to 50% of total GPU utilization 
+...
+# pod-B of UserB
+      resources:
+        limits:
+          nvidia.com/gpu: 2 # requesting 2 vGPUs
```

Comment:
nvidia.com/gpu Declare how many physical GPUs the pod needs,

nvidia.com/gpumem, nvidia.com/gpucores Identifies how much GPU memory and GPU core each physical GPU allocates to the pod